AVInput/CHANGELOG.md

```diff
@@ -16,10 +16,14 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.0.1] - 2022-10-10
+### Added
+- Combine HdmiInput and CompositeInput Plugins into AVInput
```

Comment:
This should be a minor version update since it's adding new functionality. Hope there is no breaking change.